Makefile

```diff
@@ -103,8 +103,13 @@ test: vendor $(GOPATH)/src/$(ORG)
 	@go test -v $(shell $(PACKAGES))
 
 .PHONY: integration
+ifdef BINPATH
```

Comment:
I think this can be solved more elegantly by introducing a variable 'MINISHIFT_INTEGRSTION_TEST_BINARY' (be clear with the naming).

We then can define at the top of the Makefile after the respective use variables are defined:

    MINISHIFT_INTEGRSTION_TEST_BINARY ?= $(BUILD_DIR)/$(GOOS)-$(GOARCH)/minishift$(IS_EXE)

Then instead of an if statement in the integration target we have:

    go test -timeout 3600s $(REPOPATH)/test/integration --tags=integration -v -args --binary $(MINISHIFT_INTEGRSTION_TEST_BINARY)

The '?=' in the variable declaration will make sure that the default is used if the varible is not defined on the command line, otherwise the command line value is used.